ts/features/fci/screens/valid/FciDataSharingScreen.tsx

```diff
@@ -76,109 +61,163 @@ const FciDataSharingScreen = (): React.ReactElement => {
   );
   const email = useIOSelector(profileEmailSelector);
 
+  const startSupportRequest = useStartSupportRequest({
+    contextualHelp: emptyContextualHelp
+  });
+
+  React.useLayoutEffect(() => {
```

Comment:
to avoid the repetition of this code in multiple files we can use the hook useHeaderSecondLevel (Ref: https://github.com/pagopa/io-app/blob/master/ts/hooks/useHeaderSecondLevel.tsx#L61)

Reply:
https://github.com/pagopa/io-app/pull/5377/commits/6229773c6dc14b7e42805bae8ec434821f2ea570